### DISCUSSION TITLE
Light Sensor

### DISCUSSION THREAD

**Discussion by dredgy** [iklein99/homebridge-smartthings#264] [Ideas]
Hey just checking in to understand why the light sensor comes through as a switch as it can have more than one value.

No big deal, but it would be cool for me if I could use my TV's light sensor, which is very well-positioned in my room. Though I know Homekit makes it very difficult to automate with light sensors.

**Comment by kshartman** [comment]
Probably because it's implemented as a switch rather than a sensor.  There are a variety of HomeKit sensor characteristics that would probably work (see https://developer.apple.com/documentation/homekit/characteristic-types).  What does the sensor look like or do in smart things?
